CMakeLists.txt

```diff
@@ -229,8 +229,14 @@ if(${OMP})
     find_package(OpenMP REQUIRED)
 endif()
 
+if(${SMP})
```

Comment:
I think the preferred style of using CMake variables in if statements is to leave out the `${}`, in this case, `if(SMP)`. We've been doing it this way in new PRs.

Reply:
Okay, fixed in the latest build.